geozero/src/csv/csv_error.rs

```diff
@@ -0,0 +1,14 @@
+//! CSV Error type.
```

Comment:
i usually just add the doc string to `CsvError` - doing it module level seem a bit weird tbh, esp because the module is not public - only the enum is.